src/topp/PercolatorAdapter.cpp

```diff
@@ -303,8 +303,6 @@ class PercolatorAdapter :
     registerInputFile_("fasta", "<file>", "", "Provide the fasta file as the argument to this flag, which will be used for protein grouping based on an in-silico digest (only valid if option -protein_level_fdrs is active).", !is_required, is_advanced_option);
     setValidFormats_("fasta", ListUtils::create<String>("FASTA"));
     registerStringOption_("decoy_pattern", "<value>", "random", "Define the text pattern to identify the decoy proteins and/or PSMs, set this up if the label that identifies the decoys in the database is not the default (Only valid if option -protein_level_fdrs is active).", !is_required, is_advanced_option);
-    registerStringOption_("post_processing_tdc", "<value>", "true", "Use target-decoy competition to assign q-values and PEPs.", !is_required, is_advanced_option);
```

Comment:
this changes the default from using target decoy competition to not using it!

Reply:
my guess is that it was changed to string to make it default and then some merge got the flag back in

Reply:
no it does not.
The current (and old!) behavior is that the default is dictated by the `post_processing_tdc` flag (which is `false`).
So my fix merely makes the default behaviour more clear.
The default value of the string option was ignored -- this never worked as far as I can tell.

Reply:
hmm right. but seems that intention was to make it default:
https://github.com/OpenMS/OpenMS/pull/4895
https://github.com/OpenMS/OpenMS/pull/3646

I am just a bit worried that this could break some workflows that already exist

Reply:
should not break anything since passing
` -post_processing_tdc true` on the current develop (before this PR) yields

`Trailing text argument(s) '[true]' given. Aborting!`

This means the string param could not have been used by anyone.
If someone set the parameter in an INI file, it would not lead to a crash, but I'd be suprised if the value made any difference.